docs/framework/network-programming/how-to-configure-network-tracing.md

```diff
@@ -72,25 +73,25 @@ The application or computer configuration file holds the settings that determine
     </configuration>  
     ```  
   
- When you add a name to the `<switches>` block, the trace output includes information from some methods related to the name. The following table describes the output.  
+When you add a name to the `<switches>` block, the trace output includes information from some methods related to the name. The following table describes the output.
   
 |Name|Output from|  
 |----------|-----------------|  
-|`System.Net.Sockets`|Some public methods of the <xref:System.Net.Sockets.Socket>, <xref:System.Net.Sockets.TcpListener>, <xref:System.Net.Sockets.TcpClient>, and <xref:System.Net.Dns> classes|  
-|`System.Net`|Some public methods of the <xref:System.Net.HttpWebRequest>, <xref:System.Net.HttpWebResponse>, <xref:System.Net.FtpWebRequest>, and <xref:System.Net.FtpWebResponse> classes, and SSL debug information (invalid certificates, missing issuers list, and client certificate errors.)|  
-|`System.Net.HttpListener`|Some public methods of the <xref:System.Net.HttpListener>, <xref:System.Net.HttpListenerRequest>, and <xref:System.Net.HttpListenerResponse> classes.|  
-|`System.Net.Cache`|Some private and internal methods in `System.Net.Cache`.|  
-|`System.Net.Http`|Some public methods of the  <xref:System.Net.Http.HttpClient>,  <xref:System.Net.Http.DelegatingHandler>,  <xref:System.Net.Http.HttpClientHandler>, <xref:System.Net.Http.HttpMessageHandler>,  <xref:System.Net.Http.MessageProcessingHandler>, and  <xref:System.Net.Http.WebRequestHandler> classes.|  
-|`System.Net.WebSockets.WebSocket`|Some public methods of the <xref:System.Net.WebSockets.ClientWebSocket> and <xref:System.Net.WebSockets.WebSocket> classes.|  
-  
- The attributes listed in the following table configure trace output.  
-  
+|<xref:System.Net.Sockets>|Some public methods of the <xref:System.Net.Sockets.Socket>, <xref:System.Net.Sockets.TcpListener>, <xref:System.Net.Sockets.TcpClient>, and <xref:System.Net.Dns> classes|  
```

Comment:
I don't think the values in the "Name" column should be xref'd because it's trying to show the string you'd put in the config file.